src/main/sensors/pitotmeter.c

```diff
@@ -0,0 +1,150 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <math.h>
+
+#include "platform.h"
+
+#include "common/maths.h"
+
+#include "drivers/pitotmeter.h"
+#include "config/config.h"
+
+#include "sensors/pitotmeter.h"
+
+extern int16_t debug[4];
+
+pitot_t pitot;                  // pitotmeter access functions
+uint16_t calibratingP = 0;      // baro calibration = get new ground pressure value
+float pitotPressure = 0;
+float pitotTemperature = 0;
+float CalibratedAirspeed = 0;
+int32_t AirSpeed = 0;
```

Comment:
All above variables except `pitot` and `Airspeed` should be `static` and inside `#ifdef PITOT` build flag.